examples/bezier/core.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+	"image/color"
+	"strconv"
+
+	"github.com/oakmound/oak"
+	"github.com/oakmound/oak/event"
+	"github.com/oakmound/oak/mouse"
+	"github.com/oakmound/oak/render"
+	"github.com/oakmound/oak/shape"
+)
+
+var (
+	cmp *render.Composite
+)
+
+func renderCurve(floats []float64) {
+	bz, err := shape.BezierCurve(floats...)
+	if err != nil {
+		fmt.Println(err)
+	}
+	if cmp != nil {
+		cmp.UnDraw()
+	}
+	cmp = bezierDraw(bz)
+	render.Draw(cmp, 0)
+}
+
+func main() {
+
+	// c bezier X Y X Y X Y ...
+	// for defining custom points without using the mouse.
+	// does not interact with the mouse points tracked through left clicks.
+	oak.AddCommand("bezier", func(tokens []string) {
+		if len(tokens) < 4 {
+			return
+		}
+		tokens = tokens[1:]
+		var err error
+		floats := make([]float64, len(tokens))
+		for i, s := range tokens {
+			floats[i], err = strconv.ParseFloat(s, 64)
+			if err != nil {
+				fmt.Println(err)
```

Comment:
Should this be fmt over dlog?

Reply:
I don't think it's important to use dlog in the examples, since the examples so far don't set any config in the first place.